htdocs/societe/class/societe.class.php

```diff
@@ -1936,6 +1947,12 @@ function add_commercial(User $user, $commid)
 	 */
 	function del_commercial(User $user, $commid)
 	{
+		$error=0;
+		$this->commercial_modified = $commid;
```

Comment:
Parameters to a triggers must be passed with
$this->context=array('commercial_modified'=>$commid);